docs/Using Fleet/MDM-Windows-setup.md

```diff
@@ -0,0 +1,69 @@
+# Windows setup
+
+## Supported Windows versions
+
+Windows 10 (Pro) and higher.
+
+## Overview
+
+> Windows MDM features are not ready for production and are currently in development. These features are disabled by default.
+
+Windows MDM features require configuring Fleet with a certificate and key. This guide will walk you through how to upload these to Fleet and turn on Windows MDM in order to use Windows MDM features.
+
+## Requirements
+To use Fleet's Windows MDM features you need to have:
+- A [deployed Fleet instance](../Deploying/Introduction.md).
+- A Fleet user with the admin role.
+
+## Configuring Fleet
+
+Fleet uses a certificate and key pair to authenticate and manage interactions between Fleet and Windows host.
+
+This section will show you how to:
+1. Configure Fleet with your certificate and key
+2. Turn on Windows MDM in Fleet
+
+### Step 1: configure Fleet with your certificate and key
+
+1. In your Fleet server configuration, set the contents of the certificate and key in the following environment variables:
+
+> Note: Any environment variable that ends in `_BYTES` expects the file's actual content to be passed in, not a path to the file. If you want to pass in a file path, remove the `_BYTES` suffix from the environment variable.
+
+- [FLEET_MDM_WINDOWS_WSTEP_IDENTITY_CERT_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-windows-wstep-identity-cert-bytes)
```

Comment:
@marcosd4h these can be set to any valid certificate/key pair right? If so, what's the easiest way to generate a valid cert/key pair? 

This way, we can add instructions for our users to go to this cert generator service to make their cert/key.

Reply:
> these can be set to any valid certificate/key pair right?

Marcos: Yes. We could instruct the user to generate an SSL cert.

Noah: For now, we'll instruct the user to either reuse the SCEP cert/key (if they're using macOS MDM) or run the `fleetctl generate apple-mdm` command. While the second part is potentially confusing, it's easier than generating a generic SSL cert.

Noah: This is a short term solution. Long term, we want to auto-generate and store this cert/key pair for the user (story is [here](https://github.com/fleetdm/fleet/issues/13668))